docs/containers-auth.json.5.md

```diff
@@ -37,8 +37,28 @@ their accounts on quay.io and docker.io:
 An entry can be removed by using a `logout` command from a container
 tool such as `podman logout` or `buildah logout`.
 
+In addition, credential helpers can be configured for specific registries and the credentials-helper
+software can be used to manage the credentials in a more secure way than depending on the base64 encoded authentication
+provided by `login`.  If the credential helpers are configured for specific registries, the base64 encoded will not be used
+for operations concerning credentials of the specified registries.
+
+When the credential helper is in use on a Linux platform, the auth.json file would contain keys specify the registry domain, and values specify the suffix of the program to use (i.e. everything after docker-credential-).  For example:
+
+```
+{
+    "auths": {
+        "localhost:5001": {}
+    },
+    "credHelpers": {
+		"registry.example.com": "secretservice"
+	}
+}
+```
+
+For more information on credential helpers, please reference the [GitHub docker-credential-helpers project](https://github.com/docker/docker-credential-helpers/releases).
+
 # SEE ALSO
-    buildah-login(1), buildah-logout(1), podman-login(1), podman-logout(1)
+    buildah-login(1), buildah-logout(1), podman-login(1), podman-logout(1), skopeo-login(1), skopeo-logout(1)
 
```

Comment:
I know it's old, but we only list podman and bulidah login at the top.  I'd suggest: changing line 15->17 to:

```
The file can have zero to many entries and
is created by a `login` command from a container tool such as `podman login`, 
`buildah login` or `skopeo login`.  
```

Reply:
fixed